particles/Chooser/Chooser.js

```diff
@@ -10,14 +10,55 @@
 
 "use strict";
 
-var host = `[chooser]`;
-var productStyles = '';
+defineParticle(({DomParticle}) => {
 
-importScripts('../../../particles/shared/product-templates.js');
+  const host = `[chooser]`;
 
-defineParticle(({DomParticle}) => {
+  const productStyles = `
```

Comment:
It's not that there's "no way", it's just that the simplistic method isn't sufficient, and I triaged this for postponement.

Fwiw, this is not just about stylesheets, but ultimately about having robust composition on various axes (templates, classes, style, so on).